docs/eventing/getting-started.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Getting Started with Eventing"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 9
+type: "docs"
+---
+
+This getting started guide will walk you through all the steps involved in a typical Knative use case. You will learn how to create, send, and verify events in Knative.
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative Eventing installed](../install/README.md). 
+
+## Setting up Eventing subcomponents
+
+Before you start to send events, you need to create the components needed to transport the events. This guide walks you through creating each Eventing subcomponent.
+
+### Configuring namespace for eventing
```

Comment:
After performing the steps I think the title should change a bit to align with the actual task being performed.

```suggestion
### Creating and configuring an Eventing namespace
```

@grantr is it correct to define "an Eventing namespace" as a namespace that has the `knative-eventing-injection` label?

Reply:
:+1: For this document "Eventing namespace" is a reasonable shorthand.

![technically-correct](https://user-images.githubusercontent.com/680/62226566-3a28b980-b36f-11e9-8cf9-36ffc472003f.png)